bin/xcode-build-phase/increment-build-number.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+HEADER="${PROJECT_DIR}/${INFOPLIST_PREFIX_HEADER}"
```

Comment:
@jmoody Where do you get these ENV vars from? I'm not doubting them, I've just always wanted to know :)